### ISSUE TITLE
There should be aseparator line between 'Delete Permanently' and 'Restore'

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/nemo#3325]
### Distribution

Mint 21.2

### Package version

5.8.5

### Frequency

Always

### Bug description

Every so often accidentally I delete an item from the bin rather than restoring it or else vice-versa. Such a mistake can be consequential. Such a mistake is easy to make especially with imperfect input devices.

A solution is easy: put a separator line between the two items. This is low-hanging but important fruit, as it were. (Mint is meant to be user-friendly.)

### Steps to reproduce

1) Move something to the bin.
2) Go to either delete the item permanently or either restore it.
3) If you make a mistake - is your touchpad bad? - regret 2.

### Expected behavior

I expect (i.e. think there should be) a separator line.

### Additional information

[In 2019 I raised this problem as a feature request (#2101). Now though it seems to me better categorised as a bug. To that feature request I added a request for pointers on how to fix the code myself. I received no response to that latter request.

**Comment by Jeremy7701** [comment]
You would only go to the bin if you had inadvertently deleted something.
This should be an extremely rare occurrence!

You shouldn't treat the bin as a safe haven to store data in - if you need a temporary holding area there are several better solutions.

**Comment by LinuxOnTheDesktop** [comment]
@Jeremy7701

> You would only go to the bin if you had inadvertently deleted something.

Another reason to go to the bin: to see whether it is safe to empty the bin and thereby save some space.

> You shouldn't treat the bin as a safe haven to store data in

The bin is at least a kind of fail-safe. That is its very point.

More: suppose that I am wrong about all of some of this; how would that undermine the idea that a separator would be good?

**Comment by Jeremy7701** [comment]
Obviously this has no bearing on whether a separator might be good - although I have noticed that the delete and bin options are already enclosed by separator lines, so that the two delete options are separated from other other options.

![Screenshot](https://github.com/linuxmint/nemo/assets/52585119/f8aaf78d-aee6-48a4-9808-3d43eb5953ec)

**Comment by LinuxOnTheDesktop** [comment]
That the _two_ options are, as a group, enclosed within separators does encourage one to pay attention to the enclosed options. Yet, it remains easy to select the wrong one. Note also that, as your screenshot shows (and my system confirms), _the separator bars are nearly invisible._ That is a problem in itself.

**Comment by AlexFolland** [comment]
I agree with the logic of the rational user here.  Having a space of some kind between "Delete Permanently" and "Restore" is obviously an improvement in user-friendliness with no downside.  It is counterproductive to argue against it.  Instead, best to just accept a pull request and otherwise leave the ticket instead of arguing against something that's obviously an improvement.

**Comment by LinuxOnTheDesktop** [comment]
Here is how Windows (Windows 10) does it - sensibly, with horizontal demarcation lines, and indeed an intervening option, between the two very different options that are 'restore' and 'delete [permanently]':

![image](https://github.com/user-attachments/assets/cc3cdc99-44cf-471f-b782-2e50fac3c203)

**Comment by LinuxOnTheDesktop** [comment]
It would be nice were Mint 21.1 to introduce this sane and sanity-saving little separator line . .  (which is requested not only here but also in #2101 and, by someone else, in #2972; and I seem to recall that someone else made the same request somewhere else, too).

**Comment by LinuxOnTheDesktop** [comment]
@mtwebster : please fix this.
